### PR TITLE
new image: vendors-sdk

### PR DESCRIPTION
**Pull request by ilappe**
## Related Issues
Related: https://jira-hq.paloaltonetworks.local/browse/CIAC-861

## Content PRs:
https://github.com/demisto/content/pull/29575

## Description
New image grouping the following vendor's sdk images:
```
hashicorp
cloaken
fp-smc
confluent-kafka
duoadmin3
opencti-v4
cymruwhois
joe-security
```
 - **131 MB size**

should be used in the following yml files:
```
Packs/HashiCorp-Vault/Integrations/HashiCorpVault/HashiCorpVault.yml
Packs/Cloaken/Integrations/Cloaken/Cloaken.yml
Packs/ForcepointSecurityManagementCenter/Integrations/ForcepointSecurityManagementCenter/ForcepointSecurityManagementCenter.yml
Packs/Kafka/Integrations/KafkaV3/KafkaV3.yml
Packs/DuoAdminApi/Integrations/DuoAdminApi/DuoAdminApi.yml
Packs/DuoAdminApi/Integrations/DuoEventCollector/DuoEventCollector.yml
Packs/OpenCTI/Integrations/OpenCTI/OpenCTI.yml
Packs/FeedOpenCTI/Integrations/FeedOpenCTI_v4/FeedOpenCTI_v4.yml
Packs/TeamCymru/Integrations/TeamCymru/TeamCymru.yml
Packs/JoeSecurity/Integrations/JoeSecurityV2/JoeSecurityV2.yml
```

image

### CODE REVIEW

**Review by GuyAfik — COMMENTED** [review]
@ilappe incredible wor!, make sure you add `veirfy.py` to make sure that all the SDKs are installed correctly.

wait for @JudahSchwartz 's approval before merging also

**Comment by JudahSchwartz** [comment]
all of these packs are xsoar supported?

**Comment by ilappe** [comment]
@JudahSchwartz 
> all of these packs are xsoar supported?

as you can see in the attached table - yes 
image

**Comment by ilappe** [comment]
@JudahSchwartz 
there is additional candidate images 
but they are partner supported
```
octoxlabs
sixgill
illumio
opnsense
greynoise
gdetect
reversinglabs-sdk-py3
```
image

File: docker/vendors-sdk/pyproject.toml
Comment (JudahSchwartz) [inline docker/vendors-sdk/pyproject.toml:11]: please add a comment why this is locked. If it needs to be locked make sure to ignore it in the dependabot config also otherwise dependabot will update it anyway

**Review by JudahSchwartz — APPROVED** [review]
Looks great! Please check my comment before merge

**Review by GuyAfik — APPROVED** [review]
@ilappe champ, amazing work

**Pull request merged**
